2023/README.md

```diff
@@ -15,19 +13,20 @@ If you would like to make a proposal or contribute in other ways, read on.
 
 ## Making a proposal
 
-**Interop 2023 has not yet launched and this documentation is still being finalized. Please do not submit proposals yet.**
-
 If you've had problems using a feature on the web because of differences between browsers, or because it isn't implemented in all browsers, it may be a good proposal for Interop 2023!
 
 Before making a proposal, here's what to expect:
 
-- Only features which have high quality specifications and tests are in scope. Interop 2023 is not a venue for specifying new features; that work happens in working groups within organizations such as W3C and WHATWG.
+- There are two kinds of proposals:
+  - **Focus Area**: Most proposals should fall into the category, where the goal is for browsers to pass a set of tests for a feature. Only features which have high quality specifications and tests are in scope.
+  - **Investigation Effort**: In cases where there aren't high quality specifications and tests, an investigation effort may be appropriate. An investigation efforts consists of a set of tasks that will bring the feature up to the bar required for a Focus Area in the future. If this involves any standards work, that work must be done in the appropriate standards group, and those doing the work need to join those groups.
```

Comment:
Again, I think we should make it clearer that Investigate efforts are for areas with known interop problems, not a venue for progressing early-stage standards work.

Reply:
Also updated.